third_party/openthread/src/ncp/spinel.h

```diff
@@ -979,6 +979,9 @@ typedef enum
      */
     SPINEL_PROP_IPV6_ICMP_PING_OFFLOAD  = SPINEL_PROP_IPV6__BEGIN + 5, ///< [b]
 
+    SPINEL_PROP_IPV6_MULTICAST_ADDRESS_TABLE
```

Comment:
You remember it correctly. I considered this as well, but realized that this requires some bigger change in how OT stores/manages the addresses.